xml/cha_administration.xml

```diff
@@ -584,4 +584,119 @@
+        </step>
+        <step>
+          <para>
+            Activate the module as instructed:
+          </para>
+          <screen>&prompt.sudo;<command>SUSEConnect -p SLES-LTSS/<replaceable>&productnumber-regurl;</replaceable>/<replaceable>x86_64</replaceable> -r <replaceable>REGISTRATION_CODE</replaceable></command></screen>
+        </step>
+      </procedure>
+    </sect2>
+    <sect2 xml:id="ltss-payg">
+      <title>LTSS on PAYG</title>
+      <para>
+        LTSS subscriptions for PAYG can be transacted through a private offer on the CSPs market
+        place or via direct transaction with SUSE. Reach out to <link
+        xlink:href="mailto:cloudsales@suse.com">cloudsales@suse.com</link> to work out the
+        commercial details.  You will receive a subscription and access to the <link
+          xlink:href="&sccurl;">&scc;</link>). With your subscription, you can activate a
+        registration code for LTSS.
+      </para>
+      <tip>
+        <title>Existing LTSS subscriptions</title>
+        <para>
+          If you already have an LTSS subscription that you are using in your data center, it will
+          work in the cloud just fine. You can deregister a system in your data center and move that
+          use of your LTSS subscription to an instance in the cloud.
+        </para>
+      </tip>
+      <procedure>
+        <para>
+          To enable the LTSS extension, perform the following steps:
+        </para>
+        <step>
+          <para>
+            Log in to the <link xlink:href="&sccurl;">&scc;</link> to activate a registration code.
+            Note that LTSS subscriptions are version-specific. If you have a subscription for
+            &slsa;&nbsp;15&nbsp;SP4, you cannot use that registration code to register LTSS on a
+            15&nbsp;SP3&nbsp;image. <emphasis>Make sure to activate the LTSS registration code for
+            the correct version and service pack (SP) of your instance!</emphasis>
+          </para>
+        </step>
+        <step>
+          <para>
+            Log in to your instance and make sure your system is registered with a subscription that
+            is eligible for LTSS. If the system is not yet registered, register it
+            (see <xref linkend="sec-admin-register"/>).
+          </para>
+        </step>
+        <step>
+          <para>
+            Log in to your instance and update <package>cloud-regionsrv-client</package> to the
+            latest version.
+          </para>
+          <screen>&prompt.sudo;zypper up cloud-regionsrv-client</screen>
+          <para>
+            You will need at least version 10.3.4.
+          </para>
+        </step>
+        <step>
+          <para>
+            Register the LTSS extension:
+          </para>
+<screen>&prompt.sudo;registercloudguset -r $REGISTRATION_CODE
+Running LTSS registration...this takes a little longer
+LTSS registration succeeded</screen>
```

Comment:
Should we have a side note here to clarify that `$REGISTRATION_CODE` is the LTSS registration code previously activated in SCC?